paths.json

```diff
@@ -14696,6 +14696,18 @@
     "results/voyageai__voyage-3/no_revision_available/LegalQuAD.json",
     "results/voyageai__voyage-3/no_revision_available/XPQARetrieval.json"
   ],
+  "voyageai__voyage-code-3": [
```

Comment:
Should we create CI to auto-update this?

Reply:
I don't think that we need, because soon will be `leaderboard2.0`

Reply:
I thought this was also used for the HF dataset?

Reply:
For HF dataset yes, but I'm not shure if it popular, because mteb get results by cloning repository, but if you want I can add CI